lib/std/fmt/parse_float/parse.zig

```diff
@@ -156,7 +156,7 @@ fn parsePartialNumberBase(comptime T: type, stream: *FloatStream, negative: bool
         };
     }
 
-    n_digits -= info.max_mantissa_digits;
+    n_digits -= @as(isize, info.max_mantissa_digits);
```

Comment:
This is incorrect because `info` is comptime-known.

Reply:
See above